trace/config.go

```diff
@@ -24,8 +24,10 @@ type Config struct {
 	DefaultSampler Sampler
 }
 
-// SetConfig sets the global tracing configuration.
-func SetConfig(cfg Config) {
+// ApplyConfig applies changes to the global tracing configuration.
+//
+// Fields not provided in the given config is going to be preserved.
+func ApplyConfig(cfg Config) {
 	if cfg.DefaultSampler == nil {
 		cfg.DefaultSampler = newDefaultSampler()
```

Comment:
Here you don't want to reinstall the default, you want to keep the previous value. Maybe worth having a test.

Reply:
Done.